apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/SpringConfiguration.java

```diff
@@ -64,6 +64,9 @@ public class SpringConfiguration {
   public static final String PROP_DB_PASSWORD = "bfdServer.db.password";
   public static final String PROP_DB_CONNECTIONS_MAX = "bfdServer.db.connections.max";
   public static final String PROP_DB_SCHEMA_APPLY = "bfdServer.db.schema.apply";
+  /** The {@link String} for the parameter to use the fake drug code implementation */
```

Comment:
I suggest a bit more detail here about the parameter. Something along these lines:

Boolean property that is used to enable the fake drug code (00000-0000) that is used for integration testing. When this property is set to the string 'true', this fake drug code will be appended to the drug code lookup map to avoid test failures that result from unexpected changes to the external drug code file <<ideally put a link here to the relevant code that reads that file and generates the map>>. This property defaults to false and should only be set to true when the server is under test in a local environment.

Reply:
done